main.js

```diff
@@ -0,0 +1,96 @@
+var Board = function(numPegs=3, numDiscs=5) {
+  this.numPegs = numPegs,
+  this.numDiscs = numDiscs,
+  this.layout = [];
+  for (let index = 0; index < numPegs; index++) {
+    this.layout.push([]);    
+  };
+  for (let index = 0; index < numDiscs; index++) {
+    this.layout[0].push(index + 1);    
+  };  
+  
+}; //change so that user can specify number of pegs.
+
+var board = new Board;
+
+Board.prototype.printBoard = function() {
+  this.layout.map(function (peg) {      
+    console.log('--- ' + peg.join(' '));
```

Comment:
`map` should return an array, you're essentially using `map` as if it were `forEach` here.